spring-boot/admin/src/main/java/eu/coatrack/admin/model/repository/MetricsAggregationCustomRepository.java

```diff
@@ -379,6 +376,28 @@ public long getNumberOfApiCallers(LocalDate from, LocalDate until, String apiPro
         return getSummarizedMetrics(serviceId, null, fromDate, untilDate);
     }
 
+    public List<StatisticsPerService> getStatisticsPerServiceByServicesOwnedByLoggedInUserAndDateRange(
```

Comment:
Maybe `getStatisticsOfServicesOwnedByLoggedInUserAndByDateRange` would be a better name.

Reply:
Done